### ISSUE TITLE
Ontrack Change Log plugin must not synchronize on the build

### ISSUE THREAD

**Issue by dcoraboeuf** [jenkinsci/ontrack-plugin#26]
The Ontrack Change Log plugin currently synchronizes on the previous build:

```java
@Override
public BuildStepMonitor getRequiredMonitorService() {
    return BuildStepMonitor.BUILD;
}
```

In case of concurrent builds, this causes the step to wait for the previous build to complete.

According to the documentation] at http://javadoc.jenkins-ci.org/hudson/tasks/BuildStep.html#getRequiredMonitorService(), the `NONE` build monitor should be enough.

**Comment by dcoraboeuf** [comment]
Since the Ontrack Change Log plug-ins relies on the parameter actions to be filled in, we might have to rely on the `CheckPoint.block()` method to wait for them to be available.

**Comment by dcoraboeuf** [comment]
Will be available in 2.29.1
